src/libs/OptionsListUtils.js

```diff
@@ -204,20 +205,76 @@ function hasReportDraftComment(report) {
         && lodashGet(reportsWithDraft, `${ONYXKEYS.COLLECTION.REPORTS_WITH_DRAFT}${report.reportID}`, false);
 }
 
+/**
+ * @param {Object} report
+ * @returns {Array<String>}
+ */
+function getParticipants(report) {
+    const participants = [...lodashGet(report, ['participants'], [])];
+    if (ReportUtils.isPolicyExpenseChat(report)) {
+        if (!_.contains(participants, report.ownerEmail)) {
+            participants.push(report.ownerEmail);
+        }
+    }
+    return participants;
+}
+
+/**
+ *  Get the Avatar url or fallback to the default icon according to the chat type
+ *
+ * @param {String} source
+ * @param {Object} options
+ * @param {Boolean} [options.isChatRoom]
+ * @param {Boolean} [options.isArchivedRoom]
+ * @param {Boolean} [options.isPolicyExpenseChat]
+ * @returns {String | function():any}
+ */
+function getAvatarSource(source, {isChatRoom, isArchivedRoom, isPolicyExpenseChat}) {
```

Comment:
NAB, I think this would be better if we pass a type as a string instead of an object with the different boolean props (they look like they are mutually exclusive so not sure why we would pass an object of one or more of them).